how-to/sharing_docs.md

```diff
@@ -0,0 +1,22 @@
+# Sharing Docs with the Community
+
+## Prerequisites
+
+You should be a member of the [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) group in order to view docs in the shared Google Drive.
+
+## Moving Google Docs into the shared Google Drive
+
+1. Assign ownership of the Google doc to drive-content-managers@kubeflow.org ([instruction](https://support.google.com/drive/answer/2494892)).
```

Comment:
@jlewi I just realized that this instruction doesn't work, I cannot assign ownership to this group email regardless if it's a doc owned by an @google.com account or an @gmail.com account. The "Make owner" option is always grayed out.
Furthermore, it seems Google doc doesn't even allow transfer ownership to individual email address if the email belongs to a different domain.
Would you give another thought on the process? Thanks!